docs/advanced-techniques.md

```diff
@@ -85,6 +85,30 @@ A module is a standalone configuration object that can reference client code, re
 
 <!-- prettier-ignore-end -->
 
-A merged module's code is treated the same as the application's code. The resulting application's bundles include the module code and configurations as if they were always a part of the application.
+A merged module's code is included into the resulting application's bundles and behaves as a part of the application.
```

Comment:
What do you mean "and behaves"?

Reply:
The code behaves

Reply:
How can code 'behave'?

Reply:
Changed to "is interpreted as a part of the application"